examples/android/app/src/main/proto/helloworld.proto

```diff
@@ -29,6 +29,7 @@
 
 syntax = "proto3";
 
+option optimize_for = LITE_RUNTIME;
```

Comment:
Actually, I just removed it and it seems like everything is still playing nicely. Code looks like everything is in lite as well.